Zend/tests/bug79514.phpt

```diff
@@ -2,12 +2,15 @@
 Bug #79514 (Memory leaks while including unexistent file)
 --FILE--
 <?php
-$mem1 = memory_get_usage(true);
-for ($i = 0; $i < 100000; $i++) {
-    @include __DIR__ . '/bug79514.doesnotexist';
+$filename = __DIR__ . '/bug79514.doesnotexist';
+// approximate size of the zend_string
+$size = 8 + 3 * PHP_INT_SIZE + strlen($filename) & ~(PHP_INT_SIZE - 1);
+$mem1 = memory_get_usage();
+for ($i = 0; $i < 100; $i++) {
+    @include $filename;
 }
-$mem2 = memory_get_usage(true);
-var_dump($mem2 - $mem1 < 100000);
+$mem2 = memory_get_usage();
+var_dump($mem2 - $mem1 < 100 * $size);
```

Comment:
Would it be possible to do something even more aggressive here?
```
<?php
$filename = __DIR__ . '/bug79514.doesnotexist';
@include $filename;
// Further include should not increase memory usage.
$mem1 = memory_get_usage();
@include $filename;
$mem2 = memory_get_usage();
var_dump($mem1 == $mem2);
```

Reply:
For 100 iterations, there was a memory increase of 480 bytes. That might be a leak still needing to be fixed.

Reply:
Does it only happen on the first include though, or also on later ones? I'm only seeing an increase on first include.

Reply:
Ah, indeed! I'm going to update the PR this evening.